src/core/matter/mod.rs

```diff
@@ -483,6 +483,16 @@ pub trait Matter: Default {
 
         Ok(())
     }
+
+    fn raw_size(&self) -> Result<u32> {
+        let sizes = tables::sizage(&self.code())?;
+        if sizes.fs != 0 {
+            Ok(sizes.fs)
+        } else {
```

Comment:
This behaviour differs: https://github.com/WebOfTrust/keripy/blob/development/src/keri/core/coring.py#L879

Reply:
`fs` means full size which includes the code.

Reply:
Oh right, and in this case it's obviously zero. Still, you should raise an exception like KERIpy does or we may confused at behaviour later.

Reply:
Actually I see the comparison is !=, doesn't that mean that we return full size if full size exists? That can't be right, I think this must be mistranslated. It should be the else clause for the fs != 0 case, and for fs == 0 it should throw an error.

Reply:
Thanks for pointing. I added a commit with the change.